eng/native/ijw/IJW.cmake

```
@@ -32,7 +32,7 @@ if (CLR_CMAKE_HOST_WIN32)
   endfunction()
 
   # 4365 - signed/unsigned mismatch
```

Could not import member, this is an issue with IJW and 
```suggestion
  # 4365 - signed/unsigned mismatch
  # 4679 - Could not import member. This is an issue with IJW and static abstract methods in interfaces. 
```

---

cc @dotnet/interop-contrib